src/infra/docs/dev-desktop.md

```diff
@@ -186,6 +186,15 @@ It helps to check <https://packages.ubuntu.com/> first and make sure that the
 package is available for both `arm64` and `amd64` architectures. The dev
 desktops are currently running Ubuntu 22.04 LTS.
 
+## Available disk space
+
+Every user has a quota of how much disk space they can use.
+Currently, the quota is set to
+[150GB](https://github.com/rust-lang/simpleinfra/blob/8a59e4faeb75a09b072671c74a7cb70160ebef50/ansible/roles/dev-desktop/defaults/main.yml#L7).
+
+Exceeding the quota will result in the error `Disk quota exceeded (os error 122)`.
+To save disk space, you may need to remove your build directory as it tends to accumulate a lot of outdated artifacts.
```

Comment:
Suggestion: for a typical rustc contributor, they might also have a separate build dir for rust-analyzer, have run `cargo-bisect-rustc` and somehow ended up with more nightly toolchains than anticipated. So maybe something like:

> To save disk space, you may need to:
> 1. Occassionaly clear your `build/` directory to clear outdated artifacts.
> 2. Remove `build-rust-analzyer/` (or whatever it's named as) if you have a separate rust-analyzer build directory.
> 3. Check if you have unneeded toolchains via rustup with `rustup toolchain list`. `cargo-bisect-rustc` may unintentionally leave nightly toolchains behind or if you use `--preserve` explicitly.

Reply:
But yeah, this is very helpful, thanks! Otherwise it can be a bit of a head-scratcher.

Reply:
I included your suggestions, thanks ❤️

Reply:
If we have to add more suggestions in the future, it might be better to extract this list into a separate doc, since these suggestions are useful even if you are developing locally 👍

Reply:
dev-guide equivalent: https://github.com/rust-lang/rustc-dev-guide/pull/2073

Reply:
thanks. I referred to your new section!